libs/ledger-live-common/package.json

```diff
@@ -281,7 +277,7 @@
     "@types/lodash": "^4.14.179",
     "@types/node": "^18",
     "@types/object-hash": "^2.1.0",
-    "@types/react": "^17.0.53",
+    "@types/react": "17",
```

Comment:
live-common actually seems incompatible with `@types/react@18` around some specific issue around the Provider typing. so meanwhile we will force that version (not having it as a peer dep) but we will come back to this issue when we need to port to v18 everywhere.